demos/dialog/index.html

```diff
@@ -153,6 +155,30 @@
         Secondary
       </mwc-button>
     </mwc-dialog>
+
+    <mwc-button data-num="8" raised>Form Validation</mwc-button>
+    <mwc-dialog id="dialog8" heading="Form Validation">
+      <p>We can validate user input before closing our dialog.</p>
+      <mwc-textfield
+        id="dialog8-text-field"
+        type="text"
```

Comment:
nit: `type="text"` is unnecessary, since that's the default type of `<mwc-textfield>`

Reply:
done!